src/sparseml/yolov8/val.py

```diff
@@ -72,18 +71,15 @@
 )
 @click.option("--plots", default=False, is_flag=True, help="show plots during training")
 @click.option(
-    "--datasets-dir",
+    "--dataset_path",
```

Comment:
Let's use --dataset-path, making it consistent w/ other arguments (dash instead of underscore)